orchestration/orchestration.go

```diff
@@ -302,6 +330,12 @@ func (o *Orchestrator) processTaskDefinition(ctx context.Context, input *Service
 		}
 		return taskDefinition, nil
 	} else if input.TaskDefinition != nil {
+		// update tags, forcing them our Org: spinup
+		input.TaskDefinition.Tags = append(input.TaskDefinition.Tags, &ecs.Tag{
+			Key:   aws.String("spinup:tag"),
+			Value: aws.String(Org),
+		})
```

Comment:
I think if you append duplicate tags with the same key, the aws api blows up? we should do the same cleanup here

Reply:
completed.